tst/EnergyPlus/unit/Autosizing/CoolingWaterDesAirOutletTempSizing.unit.cc

```diff
@@ -54,11 +54,13 @@
 #include <EnergyPlus/DataHVACGlobals.hh>
 #include <EnergyPlus/DataSizing.hh>
 #include <EnergyPlus/Fans.hh>
+#include <EnergyPlus/FluidProperties.hh>
 
 namespace EnergyPlus {
 
 TEST_F(AutoSizingFixture, CoolingWaterDesAirOutletTempSizingGauntlet)
 {
+    state->dataFluidProps->init_state(*state);
```

Comment:
I feel like this should be moved to the AutoSizingFixture base class setUp routine.  If it is something that is going to be called by each child test that uses that fixture, let's just move it out there.  What do you think?

Reply:
The problem with fixture vs unit test is whether an idf snippet exists. If idf snippet includes FluidProperties then those properties won't get read in with a fixture call to init_state because that happens before the idf snippet is read in and processed. For this specific autosizing unit test it will likely work as you suggest since there are no idf snippets.